### PR TITLE
feat(logging): add null logger for testing environment

### LINKED ISSUE #82: Create a null logger for use in the tests
Currently the test output is interleaved with a load of settings json from all the tests that instantiate `Settings`. We should try to come up with some kind of null logger that we can specify from the tests, so that the test output is easier to read.

### PR DESCRIPTION
**Pull request by brizental**
Fixes #82 

When we are in a testing environment (which means when `NODE_ENV=test`) the logger won't show anything. 

So now, instead of having `mozlog:true | false` in the config, we have `logging: null | pretty | mozlog`.

r? @vladikoff

### CODE REVIEW

File: config/test.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "logging": "null"
```
Comment (vladikoff) [inline config/test.json]: @philbooth thoughts on the name `null` ?

File: scripts/test_with_authdb.sh [hunk 2]
```diff
@@ -17,5 +17,7 @@ fi
 
 sleep 2
 
+if [ -z "$NODE_ENV" ]; then export NODE_ENV=test; fi;
```
Comment (philbooth) [inline scripts/test_with_authdb.sh]: Tiny nit, but lets be liberal with newline characters in these scripts, we have plenty spare.

File: config/test.json (hunk 1 above)
Comment (philbooth) [reply config/test.json]: I like it.

File: scripts/test_with_authdb.sh (hunk 2 above)
Comment (brizental) [reply scripts/test_with_authdb.sh]: Done.

**Pull request merged**